src/logging.rs

```diff
@@ -60,6 +60,21 @@ impl std::convert::From<Priority> for u8 {
     }
 }
 
+impl Priority {
+    fn numeric_level(&self) -> &str {
```

Comment:
@dfreese @lucab Shouldn't we make From/Into impl out of this, so that we can convert to `&'static str` with an `.into()` call?

I guess this would add public API, so I'm not sure 🤷🏿

Reply:
I had avoided that as it'd be adding to the public API, and the conversion to the numeric value, rather than the name of the enum value, might be surprising to users.

Reply:
Good points 👍🏿